os_migrate/roles/import_flavors/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
-import_flavors_validate_file: true
+os_migrate_validate_files: true
```

Comment:
I think the os_migrate_validate_files var default should go to `roles/validate_resource_files/defaults/main.yml` (i wrote that in my previous comment too btw :) ) and then the default will be in the code base only once.